bindings/wasm/test/account.spec.ts

```diff
@@ -35,12 +35,10 @@ async function run() {
 
     expect(account.getMetadata().alias).toBe('Alice');
 
-    account.setAlias('new alias');
-    expect(account.getMetadata().alias).toBe('new alias');
```

Comment:
Are we not testing this anymore?

Reply:
it doesnt actualyl work, getting the alias is still the old one. dont think we update the metadata :(